src/rtde/rtde_client.cpp

```diff
@@ -254,7 +254,7 @@ void RTDEClient::setupOutputs(const uint16_t protocol_version)
   unsigned int num_retries = 0;
   size_t size;
   size_t written;
-  uint8_t buffer[4096];
+  uint8_t buffer[8192];
```

Comment:
I assume, you had a reason to increase this? What is it?

Reply:
> What I don't like is that we effectively have two lists of field names. Maybe we can merge the version information into the g_type_list?

I thought about that too. But 2 things come to mind:
- it makes sense for the exhaustive list of outputs to be declared in the test because this is where we should define the expectations. It could be a file in the `resources` folder but that's a detail. My point is that there would still be some duplication between the test and the code, at least for the output names.
- adding version information to `g_type_list` is only useful if we want to validate and/or update the client output recipe on-the-fly based on the robot SW version. This is probably a good idea but a new feature IMHO. I think it would make sense as a separate PR.

> I assume, you had a reason to increase this? What is it?

When running the test with the full output recipe, the message sent to the robot does not fit in 4096 bytes. 
I was also looking for a way to avoid a static buffer size. It's probably doable but would require changing a bit the serialisation methods.

Reply:
Thank you for the clarifications.

I'm not sure I fully agree with the duplicated list, though. I see the test more as a validation that all the configurable outputs are actually valid and usable with a robot version.

But I agree that it would be most useful if we would validate the list depending on the software version on-the-fly. This is something we can add later. So, I would be fine with merging this like it is now.